ajax/libs/recorderjs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "recorderjs",
+  "version": "0.1.0",
+  "description": "A plugin for recording/exporting the output of Web Audio API nodes",
+  "filename": "recorder.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mattdiamond/Recorderjs.git"
+  },
+  "author": "Matt Diamond",
```

Comment:
@extend1994 I think this part is ok. Peter said that we haven't use this field in website yet.